stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>"there are multiple rows for each map".</p><p> </p><p>Earlier I think you were using the terminology of "game" for a series of rounds played on a single map (apologies if I'm confused about the terminology). It would be good to stick to the same terminology throughout ("map" feels a bit ambiguous, given that "map name" is something else).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Good point, I switched to using "game" everywhere. Except for the data column <code>map_winner</code>, which is called that in the source data. :shrug:  </p>